registration.sample.yml

```diff
@@ -3,11 +3,17 @@ as_token: aaaaaaaaaa # This again can be a random string
 hs_token: aaaaaaaaaa # ..as can this
```

Comment:
Certainly better, can we get a changelog? :)

Reply:
would that be changelog.d/162.doc? I'm not sure what format you follow, could you add a CONTRIBUTING.md file linking to towncrier or https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog or so?

Reply:
it's in progress https://github.com/Half-Shot/matrix-hookshot/pull/134/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R56